notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb

```diff
@@ -119,7 +119,12 @@
     "os.environ['CRDS_SERVER_URL'] = 'https://hst-crds.stsci.edu'\n",
     "os.environ['CRDS_PATH'] = os.path.abspath(os.path.join('.', 'reference_files'))\n",
     "\n",
-    "subprocess.check_output('crds bestrefs --files ua0605*_c0m.fits --sync-references=1 --update-bestrefs', shell=True, stderr=subprocess.DEVNULL)\n",
+    "stdout = subprocess.check_output(\n",
+    "    'crds bestrefs --files ua0605*_c0m.fits --sync-references=1 --update-bestrefs',\n",
+    "    shell=True,\n",
```

Comment:
Is there a way to do this without `shell=True`?

Reply:
Isn't CRDS written in Python? Can't we call its Python API directly?

Reply:
CC: CRDS experts @eslavich @ jaytmiller

Reply:
@pllim May I ask: what is the issue with `shell=True`?

Reply:
https://docs.python.org/3/library/subprocess.html#security-considerations

Reply:
@pllim See https://github.com/spacetelescope/notebooks/pull/140/commits/ba41308e8058077b0bc89ed47874c940f08a363a

Reply:
I consulted with Todd, there is a Python API.  This ought to be equivalent:

```python
import crds
import glob

crds.assign_bestrefs(glob.glob("ua0605*_c0m.fits"), sync_references=True)
```

Reply:
Just out of academic curiosity, is there benefit in sorting the `glob` results first in this usage?

Reply:
@eslavich Thanks! I am trying it now. If it works - it's a great solution.

Reply:
> Just out of academic curiosity, is there benefit in sorting the glob results first in this usage? I do not see why would it matter _here_.

Reply:
Well, let's say if the bestref is usually named in a way that would appear last in listing, and the function do a simple linear search, then `sorted(..., reverse=True)` would make the search faster? Anyway, I have no idea how it works, so I am just thinking aloud. And of course, it is probably out of topic as far as this PR is concerned.

Reply:
@eslavich In this usage, do we need to set the following:

os.environ['CRDS_SERVER_URL'] = 'https://hst-crds.stsci.edu'
os.environ['CRDS_PATH'] = os.path.abspath(os.path.join('.', 'reference_files'))

Reply:
The order shouldn't matter -- the glob result is a list of datasets, and we have to make the best references determination for each of them.

Incidentally, I don't think `shell=True` is a security concern in this case.  We wouldn't want to use it in web server code (for example) that interpolates user input into a command, but here the user is already in an environment where they can execute arbitrary shell commands so it's no more unsafe than a notebook in general.

Reply:
It is not as much as a safety in this case, as to promote best practices. What if someone copy-paste what is here into server code? 😉 

And given that it is possible to call Python API directly, there is really no reason to have `shell=True` at all in the first place.

Reply:
p.s. Yeah, just ignore my comment about sorting. Sorry for the noise!

Reply:
> What if someone copy-paste what is here into server code?

Ooooo 😲